javascript-light-bulb/main.js

```diff
@@ -13,8 +13,8 @@ var backgroundOff = $background.className = 'background background-off';
 var backgroundOn = $background.className = 'background background-on';
 
 function lightSwitch(event) {
-
-  if ($light.className === lightOn) {
+  var lightStatus = $light.className;
```

Comment:
This is still reliant on the current classes applied in the DOM. You should make a boolean variable to keep track of if the light is currently on or not. `var isLightOn = true`